pkg/storage/storage.go

```diff
@@ -44,14 +44,14 @@ func (s *Storage) CloseDB() {
 }
 
 // AddNewQuery adds a new query to the db
-func (s *Storage) AddNewQuery(term string, city string, radius int, chatID int64) (*model.Query, error) {
+func (s *Storage) AddNewQuery(term string, city string, radius int, chatID int64, pricemin int, pricemax int, saletype string) (*model.Query, error) {
```

Comment:
Use Camel Case

Reply:
I change it to min_price, max_price and sale_type, since only the DB is using CamelCase as of now.
But I am not really sure about it.